bentoml/cli/serverless.py

```diff
@@ -3,135 +3,73 @@
 import subprocess
 import argparse
 
-from bentoml.utils import Path
-from ruamel.yaml import YAML
+from datetime import datetime
 from packaging import version
 
-AWS_HANDLER_PY_TEMPLATE = """\
-try:
-    import unzip_requirements:
-except ImportError:
-    pass
-import json
-
-import {class_name}
-
-bento_service = {class_name}.load()
-
-def {api_name}(event, context):
-    result = bento_service.{api_name}.handle_aws_lambda_event(event)
-
-    return result
-"""
-
-GOOGLE_MAIN_PY_TEMPLATE = """\
-import {class_name}
-
-bento_service = {class_name}.load()
-
-def {api_name}(request):
-    result = bento_service.{api_name}.handle_request(request)
-    return result
-"""
+from bentoml.cli.whichcraft import which
+from bentoml.cli.aws_lambda_template import AWS_HANDLER_PY_TEMPLATE, \
+     update_serverless_configuration_for_aws
+from bentoml.cli.gcp_function_template import GOOGLE_MAIN_PY_TEMPLATE, \
+     update_serverless_configuration_for_google
 
 default_serverless_parser = argparse.ArgumentParser()
 default_serverless_parser.add_argument('--region')
 default_serverless_parser.add_argument('--stage')
 
+SERVERLESS_PROVIDER = {
+    'aws-lambda': 'aws-python',
+    'aws-lambda-py3': 'aws-python3',
+    'gcp-function': 'google-python',
+}
+
 
 def generate_base_serverless_files(output_path, platform, name):
     subprocess.call(
         ['serverless', 'create', '--template', platform, '--path', output_path, '--name', name])
```

Comment:
can you use 'cwd' option to workaround the 'current directory' limitation that serverless cli has?

Reply:
I think that way we can avoid copying the entire directory at the end, let me know if I misunderstood this